source/partials/next_release/_release.md.erb

```diff
@@ -1,73 +1,49 @@
 <h4>UX Changes</h4>
-* <%= link_to_commit '67f1605fca894ea9855d16a283a913335200f3b2', 'Find Value Stream Map (VSM) more easily', 3705%> (<%= link_to_issue 3858%>)
+* <%= link_to_commit '0e926106003a040ad4c62bc7810df0b851a35864', 'Plugins SPA is enabled by default.', 4030%>.
```

Comment:
I don't think end users care about "SPA" — "new snappier plugin settings page" is probably what will resonate :)

Reply:
Can we also create or link to an issue (https://github.com/gocd/gocd/issues/3866) for users to report bugs and mention that in the release notes? Like we did with the quick edit page?

Reply:
I like the idea, but prefer to create a new placeholder issue. #3866 should be closed after @rajiesh confirms the page is good.

Reply:
I agree with [that](https://github.com/gocd/www.go.cd/pull/719/files#r154352332) comment, but I guess older page was snappy too. So, may be change it to something along the lines of UX. Sorry, I don't have a good text in mind.